File: src/main/java/me/spartacus04/jext/SpigotVersion.java
package me.spartacus04.jext;

import org.bukkit.plugin.java.JavaPlugin;

public class SpigotVersion {
  
  private static SpigotVersion instance;

  private String version;

  public SpigotVersion(JavaPlugin plugin) {
    this.version = plugin.getServer().getBukkitVersion().substring(0, 4);
    SpigotVersion.instance = this;
  }

  public static boolean isVersion1_16() {
    return instance.version.equals("1.16");
  }

  public static boolean isVersion1_15() {
    return instance.version.equals("1.15");
  }

  public static boolean isVersion1_14() {
    return instance.version.equals("1.14");
  }

  public static boolean isVersion1_13() {
    return instance.version.equals("1.13");
  }

}

File: src/main/java/me/spartacus04/jext/SpigotVersion.kt
package me.spartacus04.jext

import org.bukkit.plugin.java.JavaPlugin


class SpigotVersion private constructor() {
    companion object {
        var MAJORVERSION = 0
        var MINORVERSION = 0

        fun load(plugin: JavaPlugin) {
            val versionString = plugin.server.bukkitVersion.split("-")[0]

            MAJORVERSION = versionString.split(".")[1].toInt()
            try {
                MINORVERSION = versionString.split(".")[2].toInt()
            } catch (_: IndexOutOfBoundsException) { }
        }
    }
}